os_migrate/plugins/module_utils/network.py

```diff
@@ -156,3 +156,140 @@ def network_refs_from_ser(conn, ser_net):
         conn, ser_params['qos_policy_name'])
 
     return refs
+
+
+def serialize_security_group_rule(sdk_sec_rule):
+    """Serialize OpenStack SDK security group rule
+    `sdk_sec_rule` into OS-Migrate format.
+
+    Returns: Dict - OS-Migrate structure for Security group rule.
+    """
+    expected_type = openstack.network.v2.security_group_rule.SecurityGroupRule
+    if type(sdk_sec_rule) != expected_type:
+        raise exc.UnexpectedResourceType(expected_type, type(sdk_sec_rule))
+
+    resource = {}
+    params = {}
+    info = {}
+    sdk_params = {}
+
+    resource[const.RES_PARAMS] = params
+    resource[const.RES_INFO] = info
+    resource[const.RES_TYPE] = const.RES_TYPE_SECURITYGROUPRULE
+
+    params['tags'] = sorted(sdk_sec_rule['tags'])
+
+    # Decide which attrs are param and which are info
+    set_ser_params_same_name(info, sdk_sec_rule, [
+        'id',
+        'security_group_id',
+        'remote_group_id',
+        'tenant_id',
```

Comment:
tenant_id should be removed, i think the new terminology is project_id which is on the line below. It will always be the same ID so no need to serialize it twice IMO.

Reply:
done